src/modules/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  CustomEris: require('./custom-eris'),
+  Command: require('./command'),
```

Comment:
I still think Command, Await, and Replacer should be in a folder named `structures`

Reply:
we don't have "structures" in JS, we only have modules and classes, basically modules is a single module exporting a few classes, so it could be renamed, but definitely not structures. If this was a standalone framework it would likely be called `lib`